Week1/homework/js-exercises/exercise10.js

```diff
@@ -0,0 +1,11 @@
+//exercise 10
+"use strict"
+let str="";
+function print(str) {
+    console.log(str);
+}
+let a1= new Array(4);
```

Comment:
You could have also written `let a1 = [4]`. Also be explicit when it comes to variable names so for instance `let firstArrayOfNumbers = [4];`

You could have done a function that checks the length of an array, so for example:

```
function checkArrayLength(array) {
   return array.length;
}
```

Then what you could also write a function for comparing the arrays:
```
function compareArray(arr1, arr2) {
   return checkArrayLength(arr1) === checkArrayLength(arr2);
}
```

This would either return true or false that you could use in your if statement so for instance:

```
print(compareArray(firstArrayOfNumbers, secondArrayOfNumbers) ? "Same length" : "Not the same length");
```